<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaneEditorPaneClass</class>
 <widget class="QWidget" name="PlaneEditorPaneClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlaneEditorPane</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="2" column="2">
    <widget class="SkComboBox" name="categoryInput"/>
   </item>
   <item row="5" column="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="seatsInput">
        <property name="specialValueText">
         <string>?</string>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>1023</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>148</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Registration:</string>
     </property>
     <property name="buddy">
      <cstring>registrationInput</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Comm&amp;ents:</string>
     </property>
     <property name="buddy">
      <cstring>commentsInput</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>C&amp;allsign:</string>
     </property>
     <property name="buddy">
      <cstring>callsignInput</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Ca&amp;tegory:</string>
     </property>
     <property name="buddy">
      <cstring>categoryInput</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&amp;Model:</string>
     </property>
     <property name="buddy">
      <cstring>typeInput</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Cl&amp;ub:</string>
     </property>
     <property name="buddy">
      <cstring>clubInput</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>&amp;Seats:</string>
     </property>
     <property name="buddy">
      <cstring>seatsInput</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="registrationInput"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="callsignInput">
     <property name="toolTip">
      <string>&lt;html&gt;Any short name, typically one or two letters or digits. This is displayed along with the registration in the flight table.&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="commentsInput"/>
   </item>
   <item row="4" column="2">
    <widget class="SkComboBox" name="clubInput">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::NoInsert</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="SkComboBox" name="typeInput">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::NoInsert</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>&amp;Flarm ID:</string>
     </property>
     <property name="buddy">
      <cstring>flarmIdInput</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="flarmIdInput"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SkComboBox</class>
   <extends>QComboBox</extends>
   <header>src/gui/widgets/SkComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>registrationInput</tabstop>
  <tabstop>callsignInput</tabstop>
  <tabstop>categoryInput</tabstop>
  <tabstop>typeInput</tabstop>
  <tabstop>clubInput</tabstop>
  <tabstop>seatsInput</tabstop>
  <tabstop>flarmIdInput</tabstop>
  <tabstop>commentsInput</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
